src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -34,8 +34,8 @@ public class Constants {
       7L; // Don't change, should match server side
   public static final int BLOB_UPLOAD_TIMEOUT_IN_SEC = 5;
   public static final int INSERT_THROTTLE_MAX_RETRY_COUNT = 60;
-  public static final long MAX_BLOB_SIZE_IN_BYTES = 512000000L;
-  public static final long MAX_CHUNK_SIZE_IN_BYTES = 32000000L;
+  public static final long MAX_BLOB_SIZE_IN_BYTES = 256000000L;
+  public static final long MAX_CHUNK_SIZE_IN_BYTES = 16000000L;
```

Comment:
Nit, might be good to add a comment here explaining why these constants were choosen.

Reply:
do we want to mention SF's internal file setting here in public comment?

Reply:
Ehhhhh probably not, let's leave this as-is then.